agent/engine/task_manager.go

```diff
@@ -480,6 +523,17 @@ func (mtask *managedTask) isContainerFound(container *api.Container) bool {
 	return found
 }
 
+func (mtask *managedTask) isResourceFound(res taskresource.TaskResource) bool {
+	found := false
+	for _, r := range mtask.Resources {
```

Comment:
Is the `Resource` filed in `managedTask` still used?

Reply:
That shall be removed once we use `taskresource`'s `cgroup` for creation and deletion.